### PR TITLE
Extraction automation

### PR DESCRIPTION
**Pull request by shaibrown5**

### CODE REVIEW

File: .github/workflows/extraction.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Extract Info
+on:
+  schedule:
+    # runs at 6 AM UTC
+    - cron:  '0 6 * * *'
```
Comment (shaibrown5) [inline .github/workflows/extraction.yml:8]: Runs at 6 am UTC which is 9 AM Israel time. this hour was chosen by the last file uploaded on the supermarkets websites

File: superx/information_extractors/branch_info_extractor.py [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 import xml.etree.ElementTree as et
-from app import supermarket_info_dictionary, session, db
-from models import Branch
+import os
+import sys
 import logging
 import requests
 
+add_to_python_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
+sys.path.append(add_to_python_path)
```
Comment (shaibrown5) [inline superx/information_extractors/branch_info_extractor.py:10]: the ci runs on ubunto and does not recognize the path to app and models. This line adds the path to the system. same as item_info extractor

**Comment by shaibrown5** [comment]
currently this does nothing, as there is no 'if __name__ == "__main__"' in either extractor. I am trying to see if I can update the entries without deleting them -  this is to compensate for the late hour of the ci running. once done I will add

File: .github/workflows/extraction.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```
Comment (simzacks) [inline .github/workflows/extraction.yml:1]: this comment is wrong

File: superx/information_extractors/branch_info_extractor.py (hunk 2 above)
Comment (simzacks) [reply superx/information_extractors/branch_info_extractor.py:10]: that should be a comment in the code, not the PR

**Review by simzacks — CHANGES_REQUESTED** [review]

**Pull request closed without merging**